models/project/issue.go

```diff
@@ -102,7 +102,37 @@ func MoveIssuesOnProjectBoard(ctx context.Context, board *Board, sortedIssueIDs
 	})
 }
 
-func (b *Board) removeIssues(ctx context.Context) error {
-	_, err := db.GetEngine(ctx).Exec("UPDATE `project_issue` SET project_board_id = 0 WHERE project_board_id = ? ", b.ID)
+func (b *Board) moveIssuesToAnotherColumn(ctx context.Context, columnID int64) error {
+	if b.ID == columnID {
```

Comment:
also, `if b.ProjectID != getBoard(columnID).ProjectID return errNoAccess` or something like that

Reply:
f280c9b